### PR TITLE
Sp2.3 search criteria

### PR DESCRIPTION
**Pull request by anuroop15**
#### Which User story does this relate to?
-#143205311

#### Brief feature description
- Enabled county and FacilityType dropdown for added search criteria.

#### Does this PR block/effect other stories?
- No

#### Do any commands needs to be run in order to run the project with this PR?
-npm run lint
-npm run dev

#### Code coverage percent diff as a result of this PR?
-37% to 39%

#### JS errors diff as a result of this PR?
-800+ to 600

### CODE REVIEW

**Comment by anuroop15** [comment]
Updated requested changes. Please verify and merge the PR

File: app/javascript/facility/facility_address.jsx [hunk 1]
```diff
@@ -1,64 +1,66 @@
...
-                                    </p>
-                            </div>
-                            <div className="small_inner_block visit_reason">
-                                <p>LAST VISIT REASON</p>
-                                <p>{this.props.facilityData.last_visit_reason ==null ? "N/A" : this.props.facilityData.last_visit_reason}</p>
-                            </div>
-                        </div>
-                    </div>
+  render() {
+      let phoneFormat = this.props.facilityData.facility_telephone.replace(/(\d{3})(\d{3})(\d{4})/, "($1)-$2-$3");
```
Comment (ratneshraval) [inline app/javascript/facility/facility_address.jsx]: Phone formatting should be extracted out to a helper method. Something to keep in mind as TODO

Comment (lauherkcambria) [inline app/javascript/facility/facility_address.jsx]: i agree --- this leads me to ask what our general approach for javascript helpers are

Comment (lauherkcambria) [inline app/javascript/facility/facility_address.jsx]: same can be said with address creation, line 15

Comment (ratneshraval) [inline app/javascript/facility/facility_address.jsx]: Agree

Comment (lauherkcambria) [inline app/javascript/facility/facility_address.jsx]: lets add a user story TODO to extract facility helper methods and close this PR

**Comment by ratneshraval** [comment]
👍

**Pull request merged**